setup.py

```diff
@@ -18,7 +18,7 @@
 #   python setup.py install make_options="-j3"
 # which will add "-j3" option to Make commandline
 cmakeOptions = ""
-makeOptions = "install"
+makeOptions = "install -j4"
```

Comment:
We shouldn't include this by default because it can cause issues on some systems. When I build on small linux vps systems I have seen build errors that occur pretty reliably when using this option but don't show up without it.